docs/whichkey/reference.md

```diff
@@ -230,13 +230,26 @@ The transient type is being deprecated in favor of the separate command `whichke
 All current definitions of transient type are converted internally at the moment.
 - Conditional type is an experimental feature that provides conditional binding behavior. See [Extra](extra/#conditional-bindings-experimental)
 
+### DisplayOption
+
+```ts
+{
+    Hidden = "hidden"
```

Comment:
Should this be a boolean?

Reply:
This is similar to the `BindingType` above, but we only have one display option right now.

For example: one can specify `"display": "hidden"`